back-end/services/analyseproject.service.js

```diff
@@ -0,0 +1,35 @@
+import ProjectDAO from "../dao/projectsDAO.js";
```

Comment:
The file name should be analyzeProject.service.js (we're just missing a capital P :))

Reply:
Ok